usecases/config/environment.go

```diff
@@ -169,6 +169,8 @@ func FromEnv(config *Config) error {
 		}
 	}
 
+	config.Profiling.Disabled = configbase.Enabled("DISABLE_GO_PROFILING")
```

Comment:
it needs to be `configbase.Enabled(os.Getenv("DISABLE_GO_PROFILING"))` - could you please double check that it works?

Reply:
Yeah so so sorry about that - I can't believe I had missed that